src/CharterManager.sol

```diff
@@ -282,10 +289,10 @@ contract CharterManagerImp {
 
     function onVatFlux(address gemJoin, address from, address to, uint256 wad) external {}
 
-    function quit(bytes32 ilk, address dst) external {
+    function quit(bytes32 ilk, address u, address dst) external allowed(u) {
```

Comment:
Btw I think we need to add `allowed(dst)` as well. This is because you are moving collateral + debt. Even if it is after `cage` you might be moving a not skimmed and underwater CDP (for some reason) to someone's position and making that person to lose collateral.

Reply:
Right.
So this actually does not relate to this new delegation change, and has to be fixed in CropJoin as well.

Reply:
<= @hexonaut see the above comments.